src/widgets/Menu/Avatar.tsx

```diff
@@ -21,7 +21,7 @@ const Avatar: React.FC<AvatarProps> = ({ profile }) => {
   const link = profile.username ? profileLink : noProfileLink;
   const isExternal = link.startsWith("http");
   const ariaLabel = "Link to profile";
-  const icon = image ? (
+  const icon = profile.username ? (
```

Comment:
Realized this will not work as the image coming through on the FE (for example) will not be undefined